doc/man7/OSSL_PROVIDER-default.pod

```diff
@@ -223,6 +223,8 @@ The OpenSSL default provider supports these operations and algorithms:
 
 =item EC, see L<EVP_KEM-EC(7)>
 
+=item ML-KEM-768, see L<EVP_KEM-ML-KEM(7)>
```

Comment:
```suggestion
=item ML-KEM-768, see L<EVP_KEM-ML-KEM-768(7)>
```

Reply:
OK -- so basically the statement is that all these files need updating as -512 and -1024 get added?

Reply:
update done